helm/cosmo/values.yaml

```diff
@@ -98,6 +98,51 @@ cdn:
 
 # Cosmo Controlplane. For more options, please refer to the README.md
 controlplane:
+  # -- Add labels to all the deployed resources
+  commonLabels: {}
+
+  # -- Configure jobs to be executed in the control plane
+  jobs:
+    # -- Used to deactivate an organization with a reason and schedule deletion
+    deactivateOrganization:
+      # -- Enables the job to be run
+      enabled: false
+      # -- Adds custom labels to the job
+      customLabels: {}
```

Comment:
Two popints

1. I'd prefer just `labels`
2. In https://github.com/wundergraph/cosmo/issues/905 the user wanted to apply labels to jobs to avoid sidecar injection. In the current setup, he would need to copy/paste it to all jobs. It would be great if the `jobs` section would have a `commonLabels` section as well.

Reply:
Thanks for the feedback: 

1. Thought about this just kept it to note it's something additional, the default labels will still be applied, this might lead to misunderstandings. I will check how this works out with number 2. 

2. I thought about moving stuff e.g. also to the global part. Seed is not really a core function, deleting users or disabling users is part of running cosmo and therefore a job 'core' function. I will check how to split it up for easier handling. (Opinion) Even though I think it's a Helm problem and we should avoid having such logic built into the charts, it's just one of the jobs, each of those might be handled differently and you have to define this transparently for each of them.

Reply:
1. So the labels will be merged? `additionalLabels` ?
2. ok, sounds good

Reply:
Let's go like this:

```
# -- Pass additional labels to all jobs
additionalJobLabels: {}

# -- Configure jobs to be executed in the control plane
jobs:
  # -- Used to deactivate an organization with a reason and schedule deletion
  deactivateOrganization:
    ...
    # -- Adds additional labels to the job
    additionalLabels: {}
    ...
```
So it's clear it get's added.